canonical/canonical.com
# Update department pages and add latest news

## Done

- Updated department pages and added latest news module
- Included functionality for fast track and featured roles (not all departments are going to have these)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: 
   - https://canonical-com-701.demos.haus/careers/engineering
     - doc: https://docs.google.com/document/d/12GWF6eVKezvENc66p3QZ5ZEO8H2QBIlU56RPZNdCPdI/
   - https://canonical-com-701.demos.haus/careers/administration
     - doc: https://docs.google.com/document/d/1Kfkdy1rwBUP1L-ToUVvgqv8WkwEa8SurILkwJJh8u0U/ 
   - https://canonical-com-701.demos.haus/careers/finance
     - doc: https://docs.google.com/document/d/1e6A3z1KsVNyShx2mHq3lNrYkXCaHy8x8BA7qzLxeuJ4/
   - https://canonical-com-701.demos.haus/careers/legal
     - doc: https://docs.google.com/document/d/1TcKYSlfV2G9X_0l-T6rZ-CWd_XJQ_4hHQCPL2CVrCts 
   - https://canonical-com-701.demos.haus/careers/marketing
     - doc: https://docs.google.com/document/d/1larR0nWbVEqeUQbIJdgWemr7I6cqzoUhA9MD6sRQZwE
   - https://canonical-com-701.demos.haus/careers/operations
     - doc: https://docs.google.com/document/d/1zLFku18ZAMVStgkIO3bCoKEXW6ShwMEkB0w9d_nCx9U/
   - https://canonical-com-701.demos.haus/careers/people
     - doc: https://docs.google.com/document/d/1sj-WToRUa9N2c5brs8iAoOfOPHJCeHwvHByq8qcSgK4
   - https://canonical-com-701.demos.haus/careers/project-management
     - doc:  https://docs.google.com/document/d/1hc5olKZcPIHFtxYAHvYFS_c-NZIYy3ZT8WqG7r021MI/
   - https://canonical-com-701.demos.haus/careers/sales
     -  doc: https://docs.google.com/document/d/1yCukmqr7do6iJgl6hKui9lAslWJ6Gol4O2E5ctCE7PY
   - https://canonical-com-701.demos.haus/careers/support-engineering
     - doc:  https://docs.google.com/document/d/1TkePZPzjdIMRTPIep6C1f8SnTPmkKiGdMyUEPMK9rH8
   - https://canonical-com-701.demos.haus/careers/web-and-design
     - doc: https://docs.google.com/document/d/1Eq3USN12tdL1sEf0DMYxO9FAJ5Vqkrl_yeYdRsfWNww/edit
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Test that links to all roles and individual jobs work as expected

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-536
